docs/src/pages/docs/crocks/Either.md

```diff
@@ -0,0 +1,1346 @@
+---
+title: "Either"
+description: "Either Crock"
+layout: "guide"
+functions: ["firsttoeither", "lasttoeither", "maybetoeither", "resulttoeither"]
+weight: 40
+---
+
+```haskell
+Either c a = Left c | Right a
+```
+
+`Either` is the canonical `Sum` type and is the base of all other `Sum` types
+included in `crocks`, such as [`Maybe`][maybe] and [`Async`][async] to name a
+few. Defined as a tagged union type, it captures the essence of disjunction as
+it provides either a `Left` value or a `Right` value and not both.
+
+Unlike [`Maybe`][maybe], which fixes its "left side", or [`Nothing`][nothing] to
+a `()` (unit), `Either` is a functor in both it's `Left` and `Right` sides. This
+allows for the ability to vary the type on either side and is akin to the
+imperative `if...else` trees that are common in programming.
+
+Like most other types in `crocks`, `Either` has a right bias in regard to
+instance methods like `map`, `ap` and `chain`. This behavior can be used to
+formally capture Error handling as the `Left` value will be maintained and most
```

Comment:
The `most` here is a little confusing. Is it `most of the future` or `most important` or maybe I misunderstood 😄. Maybe we could remove it.